### ISSUE TITLE
4 tests failing BottombarPanel and problesmView

### ISSUE THREAD

**Issue by apupier** [jrichter1/vscode-extension-tester#10]
on Fedora 30
I launched "npm run ui-test" from terminal so my own VS Code instance should not impact the tests (right?)

```
  1) BottomBarPanel
       can be toggled open:

      AssertionError: expected false to be true
      + expected - actual

      -false
      +true
      
      at /home/apupier/git/vscode-extension-tester/test/test-project/out/test/bottomBar/bottomBarPanel-test.js:24:52
      at Generator.next (<anonymous>)
      at fulfilled (out/test/bottomBar/bottomBarPanel-test.js:5:58)
      at process._tickCallback (internal/process/next_tick.js:68:7)

  2) BottomBarPanel
       can be toggled closed:
     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/apupier/git/vscode-extension-tester/test/test-project/out/test/bottomBar/bottomBarPanel-test.js)
  

  3) BottomBarPanel
       can be maximized and restored:

      AssertionError: expected 656 to equal 336
      + expected - actual

      -656
      +336
      
      at /home/apupier/git/vscode-extension-tester/test/test-project/out/test/bottomBar/bottomBarPanel-test.js:39:35
      at Generator.next (<anonymous>)
      at fulfilled (out/test/bottomBar/bottomBarPanel-test.js:5:58)
      at process._tickCallback (internal/process/next_tick.js:68:7)

  4) ProblemsView
       get warnings works:
     AssertionError: expected [ Array(2) ] to be empty
      at /home/apupier/git/vscode-extension-tester/test/test-project/out/test/bottomBar/problemsView-test.js:46:31
      at Generator.next (<anonymous>)
      at fulfilled (out/test/bottomBar/problemsView-test.js:5:58)
      at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by jrichter1** [comment]
This is related to changes in vscode DOM, should be fixed in 2.0 branch

**Comment by jrichter1** [comment]
I've merged 2.0 into master and build is green.
